frontend/appflowy_flutter/integration_test/util/base.dart

```diff
@@ -58,17 +58,21 @@ class TestFolder {
 
 extension AppFlowyTestBase on WidgetTester {
   Future<void> initializeAppFlowy() async {
-    const MethodChannel('hotkey_manager')
-        .setMockMethodCallHandler((MethodCall methodCall) async {
-      if (methodCall.method == 'unregisterAll') {
-        // do nothing
-      }
-    });
+    TestDefaultBinaryMessengerBinding.instance.defaultBinaryMessenger
+        .setMockMethodCallHandler(
```

Comment:
> 'setMockMethodCallHandler' is deprecated and shouldn't be used. Use tester.binding.defaultBinaryMessenger.setMockMethodCallHandler or TestDefaultBinaryMessengerBinding.instance.defaultBinaryMessenger.setMockMethodCallHandler instead. Pass the channel as the first argument. This feature was deprecated after v3.9.0-19.0.pre.

Reply:
Already done, or do you prefer the other method? That requires a `WidgetTester tester` though.